loqusdb/commands/cli.py

```diff
@@ -109,10 +109,11 @@ def cli(ctx, database, username, password, authdb, port, host, uri, verbose, con
 
     ctx.obj = {}
     ctx.obj['db'] = database
-    ctx.obj['user'] = username
-    ctx.obj['password'] = password
-    ctx.obj['port'] = port
-    ctx.obj['host'] = host
+    if uri:
+        ctx.obj['uri'] = uri
+    else:
+        ctx.obj['port'] = port
```

Comment:
What about user and password?

Reply:
It seems they are not really used, if not passed as part of a uri. That is however the state of current master, so feel free to fix or leave it. We will anyway use the URI locally, so all good.